src/components/NoteExpand/CommentsBox/CommentsBox.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import styles from './CommentsBox.module.scss';
+import CommentsList from './CommentsList/CommentsList';
+import { CommentsListProps } from './CommentsList/CommentsList.types';
+import CreateComment from './CommentsList/CreateComment/CreateComment';
+
+const CommentsBox: React.FC<CommentsListProps> = ({ comments }) => (
```

Comment:
it is "prop drilling"
remove comments from this component and use useSelector hook to get comments from redux in CommentsList component